frontend/views/containers/payments/PaymentsListMenu.vue

```diff
@@ -0,0 +1,100 @@
+<template lang='pug'>
+menu-parent
+  menu-trigger.is-icon-small(:aria-label='L("Show payment menu")')
+    i.icon-ellipsis-v
+
+  menu-content.c-actions-content
+    ul
+      menu-item(
+        v-if='paymentsType === "sent" || needsIncome'
+        tag='button'
+        item-id='message'
+        icon='info'
+        @click='openModal("PaymentDetail")'
+      )
+        i18n Payment details
+
+      menu-item(
+        v-if='paymentsType === "todo"'
+        tag='button'
+        item-id='message'
+        icon='times'
+        @click='cancelPayment(payment.to, payment.hash)'
+      )
+        i18n Dismiss this payment
+
+      menu-item(
+        v-if='paymentsType === "sent"'
+        tag='button'
+        item-id='message'
+        icon='times'
+        @click='cancelPayment(payment.to, payment.hash)'
+      )
+        i18n Cancel this payment
+
+      menu-item(
+        v-if='needsIncome'
+        tag='button'
+        item-id='message'
+        icon='times'
+        @click='cancelPayment(payment.to, payment.hash)'
```

Comment:
This `payment` variable needs to be defined somewhere. Clicking this menu will result in an exception. Make sure that this PR doesn't generate any exceptions in the console by using the `fakeStore`.

If it's impossible to avoid generating exceptions in the JS methods, then comment out the offending JS code and write `// TODO: ` psuedo code about what you expect the code to do, so that you or others can then fill it out with the appropriate code.